google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -105,17 +124,31 @@ public static void beforeClass() throws IOException {
                             .setMode(Field.Mode.NULLABLE)
                             .build())))
             .build();
+    bigquery.create(tableInfo);
```

Comment:
Seems like all the 20 tables can be created with the `createTableHelper(i)` call, instead of creating the first one separately and later creating 19 more.